lms/templates/courseware/courseware-chromeless.html

```
@@ -201,4 +201,145 @@
+     *
+     * @param {HTMLElement} resizer - The element (or a descendant) within the editor.
+     * @returns {boolean}
+     */
+    const isEditorDownward = (resizer) =>
+      resizer.closest('.annotator-editor')?.classList.contains('annotator-invert-y') || false;  
+
+    /**
+     * A class that makes a <textarea> resizable by clicking and dragging an element.
+     */
+    class ResizableTextarea {
+      /**
+       * @param {HTMLElement} draggable - The element that initiates the resize on mousedown.
+       * @param {HTMLTextAreaElement} textarea - The <textarea> to be resized.
+       */
+      constructor(draggable, textarea) {
+        this.draggable = draggable;
+        this.textarea = textarea;
+        this.startX = 0;
+        this.startY = 0;
+        this.startWidth = 0;
+        this.startHeight = 0;
+
+        this.onResize = this.onResize.bind(this);
+        this.stopResize = this.stopResize.bind(this);
+
+        this.init();
+      }
+
+      /**
+       * Binds the mousedown listener on the handle in capture phase, 
+       * preventing AnnotatorJS from receiving the event first.
+       */
+      init() {
+        this.draggable.addEventListener(
+          'mousedown',
+          (event) => {
+            event.preventDefault();
+            event.stopPropagation();
+            event.stopImmediatePropagation();
+            this.startResize(event);
+          },
+          { capture: true }
+        );
+      }
+
+      /**
+       * Called on mousedown. Stores the initial position and size, then 
+       * starts listening for mousemove/mouseup at document level.
+       * 
+       * @param {MouseEvent} event - The mousedown event.
+       */
+      startResize(event) {
+        this.startX = event.clientX;
+        this.startY = event.clientY;
+        this.startWidth = this.textarea.offsetWidth;
+        this.startHeight = this.textarea.offsetHeight;
+
+        document.addEventListener('mousemove', this.onResize);
```

Why don't you just call onResize function instead of the attribute? @Serafin-dev

---

To remove event listeners you must pass the exact same function reference and we are binding both functions, and this creates a unique instance of each function, so the function and the binded one are not the same. So, if we do not do it that way, we won't be able to remove the listeners.and it will fail silently. 
Besides that, if we don't bind `this` , inside onResize, `this`loses its class context and is undefined instead.